src/scss/components/banner/_overlay-banner.scss

```diff
@@ -0,0 +1,123 @@
+@charset "UTF-8";
+
+.thub-wrapper--banner-overlay {
+    background: $su-color-black url("../assets/png/banner-bg.jpg") no-repeat 100% / cover;
```

Comment:
There is a black background attached to `.su-hero__media` for when there is no image:
<img width="1054" alt="image" src="https://github.com/SU-SWS/taiwan_hub_subtheme/assets/34019925/8917bc91-7fcc-4c84-90be-2167c356ac89">


Screen width: 1512px
![image](https://github.com/SU-SWS/taiwan_hub_subtheme/assets/34019925/acdef3ec-5aae-4db2-9741-8313592bded2)

Screen width: 650px
![image](https://github.com/SU-SWS/taiwan_hub_subtheme/assets/34019925/9ed97d44-c78d-472c-bd24-c30dd999fd57)

Reply:
Huzzah! The no-image scenario has been resolved :) Thank you @jenbreese 

![image](https://github.com/SU-SWS/taiwan_hub_subtheme/assets/34019925/6759dc32-b9a1-4214-95c9-4c798cf95394)